### ISSUE TITLE
Bug: find method does not appear to work with accession number

### ISSUE THREAD

**Issue by davelacy** [dgunning/edgartools#20]
I came across a curious situation where the `find` method does not appear to work when passing a fairly new filing accession number. 

However, finding the filing via Company method seems to fetch the filing just fine.

Example:

```python
accession_no = '0001193125-23-300021'
# not working as of 12/20/21 @ 20:43 ET
find(accession_no')

cik = '0001588272'
# vs this which works fine
Company(cik).get_filings().filter(form='8-K')[0]

```

Nothing critical just thought I'd log it! Thanks :)

**Comment by dgunning** [comment]
Thanks. Will take a look. I know the approximate reason, and should be an easy fix .. I just have to think about the API and user expectations a bit. Will tell you more later.

**Comment by dgunning** [comment]
Fixed in 3.6.4. Find will now look for the most recent filings for today

**Comment by davelacy** [comment]
Awesome! Confirmed it works. Thank you @dgunning !!! 🙏 🚀
